coreneuron/nrniv/global_vars.cpp

```diff
@@ -9,6 +9,10 @@
 #include "coreneuron/nrniv/nrniv_decl.h"
 #include "coreneuron/nrnoc/membfunc.h"
 #include "coreneuron/nrniv/nrn_assert.h"
+#include "coreneuron/nrniv/nrn2core_direct.h"
+
+void* (*nrn2core_get_global_dbl_item_)(void*, const char*& name, int& size, double*& val);
+int (*nrn2core_get_global_int_item_)(const char* name);
```

Comment:
can we transfer these to single header? (nrn2core_direct.h)

Reply:
I thought they needed a separare declaration in the header (extern) and then a definition in
some file (memory to hold the function pointer)